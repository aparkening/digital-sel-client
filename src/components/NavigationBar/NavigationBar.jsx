import React from 'react';

import Navbar from 'react-bootstrap/Navbar';
import Nav from 'react-bootstrap/Nav';
import logo from 'assets/images/moe-icon.svg';

const NavigationBar = () => (
  <header>
    <Navbar bg="white" collapseOnSelect expand="md">

      <Navbar.Brand href="/">
        <img
          alt="The Digital SEL Logo"
          src={logo}
          className="d-inline-block navbar-logo"
        />
        {' '}
        <span className="nav-brand-complete">The Digital South English Legendary</span>
        {' '}
        {/* Larger displays */}
        <span className="nav-brand-truncated">The Digital SEL</span>
        {' '}
        {/* Smaller displays */}
      </Navbar.Brand>
      <Navbar.Toggle aria-controls="basic-navbar-nav" />
      <Navbar.Collapse id="basic-navbar-nav">
        <Nav className="ml-auto">
          <Nav.Link href="#dashboard">Dashboard</Nav.Link>
          <Nav.Link href="#about">About</Nav.Link>
        </Nav>
      </Navbar.Collapse>

    </Navbar>
  </header>
);

export default NavigationBar;
