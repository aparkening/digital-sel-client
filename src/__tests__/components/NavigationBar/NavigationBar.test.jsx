import React from 'react';
import renderer from 'react-test-renderer';
import NavigationBar from 'components/NavigationBar';

it('renders the NavigationBar correctly', () => {
  const tree = renderer
    .create(<NavigationBar />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
