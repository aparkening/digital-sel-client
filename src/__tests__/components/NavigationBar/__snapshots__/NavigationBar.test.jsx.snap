// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the NavigationBar correctly 1`] = `
<header>
  <nav
    className="navbar navbar-expand-md navbar-light bg-white"
  >
    <a
      className="navbar-brand"
      href="/"
    >
      <img
        alt="The Digital SEL Logo"
        className="d-inline-block navbar-logo"
        src="test-file-stub"
      />
       
      <span
        className="nav-brand-complete"
      >
        The Digital South English Legendary
      </span>
       
      <span
        className="nav-brand-truncated"
      >
        The Digital SEL
      </span>
       
    </a>
    <button
      aria-controls="basic-navbar-nav"
      aria-label="Toggle navigation"
      className="navbar-toggler collapsed"
      onClick={[Function]}
      type="button"
    >
      <span
        className="navbar-toggler-icon"
      />
    </button>
    <div
      aria-expanded={null}
      className="navbar-collapse collapse"
      id="basic-navbar-nav"
    >
      <div
        className="ml-auto navbar-nav"
        onKeyDown={[Function]}
      >
        <a
          className="nav-link"
          data-rb-event-key="#dashboard"
          href="#dashboard"
          onClick={[Function]}
          onKeyDown={[Function]}
        >
          Dashboard
        </a>
        <a
          className="nav-link"
          data-rb-event-key="#about"
          href="#about"
          onClick={[Function]}
          onKeyDown={[Function]}
        >
          About
        </a>
      </div>
    </div>
  </nav>
</header>
`;
